uktrade/great-cms
# Update desktop and mobile styling for event listing page

This PR updates the styling on the event listing page (`/export-academy/events`) on both desktop and mobile.

It also adds functionality for filters on mobile, and logic which maintains the filter section's open/closed state between page loads using local storage.

Local storage was used rather than the django session to persist this data, because we only want to maintain the open/closed state when javascript is enabled. When it is disabled, the filters will always remain open.

To test locally
- With a recent dev database dump (to make sure you have EA events and a hero snippet)
- Go to http://greatcms.trade.great:8020/export-academy/upcoming-events (or after rebasing /export-academy/events)

### Workflow


### Reviewing help


### Housekeeping

- [ ] Ran the `make manage download_geolocation_data` command

### Merging
